app/views/media/welcome.html.erb

```diff
@@ -16,4 +16,4 @@
   <p>
   <%= (_("You may also associate one or more operating systems with this medium or alternatively set this up later on the %s page.") % link_to(_("Operating systems"), operatingsystems_path)).html_safe %>
   </p>
-</div>
+</div>
```

Comment:
looks like all that changed here was removing the newline at the end of the file?

Reply:
still all that changed is the newline here, if you aren't changing this file than don't change the whitespace as it will just confuse the history.